niceday_client/niceday_client.py

```diff
@@ -234,4 +235,32 @@ def set_tracker_reminder(self, user_id: int, tracker_name: str, reminder_title:
             "userId": str(user_id),
             "recurringSchedule": recurring_schedule
         }
-        return self._call_api('POST', url, body=body)
\ No newline at end of file
+        return self._call_api('POST', url, body=body)
+
+    def upload_file(self, user_id: int, filename: str, filepath: str, file):
+        """
+        Get smoking tracker data for specific user
+
+        Args:
+            user_id: ID of the user we want to set tracker statuses for
+            filename: The name of the file to be uploaded
+            filepath: The local path of the file to be uploaded
```

Comment:
Just checking - is `filepath` the full path to the file (i.e contains the file name) or is it the path of the directory containing that file? To me, `filepath` would mean the `filename` was already part of that path.

Reply:
You are right, `filepath` inculed also the the name of the file.  I left this as a parameter, but it would be better to extract it from the path, I agree.